webservices/common/models/presidential.py

```diff
@@ -60,23 +60,15 @@ class PresidentialBySize(db.Model):
 
 class PresidentialByState(db.Model):
 
-    __table_args__ = {'schema': 'public', 'extend_existing': True}
-    __tablename__ = 'ofec_rad_analyst_vw'
+    __table_args__ = {'schema': 'public'}
+    __tablename__ = 'ofec_presidential_by_state_vw'
 
+    # TODO: Update docstrings
```

Comment:
@fec-jli if you're going to leave docstrings for later, can you put in a follow-up documentation ticket for this work, maybe 11.5?